src/lib/loaders/loaders_with_authentication/gravity.ts

```diff
@@ -4,46 +4,108 @@ import trackedEntityLoaderFactory from "lib/loaders/loaders_with_authentication/
 export default (accessToken, userID, opts) => {
   const gravityAccessTokenLoader = () => Promise.resolve(accessToken)
   const { gravityLoaderWithAuthenticationFactory } = factories(opts)
-  const gravityLoader = gravityLoaderWithAuthenticationFactory(gravityAccessTokenLoader)
+  const gravityLoader = gravityLoaderWithAuthenticationFactory(
+    gravityAccessTokenLoader
+  )
 
   return {
+    createAccountRequestLoader: gravityLoader(
```

Comment:
Just so I understand, this is "authenticated" in the sense that an access token will be passed through if the user is logged in (but not otherwise), right?

Reply:
Not quite, these are all the loaders which exist when an access token is provided to Metaphysics. All of these will pass that through to Gravity, and will also skip any caching.

The `loaders_without_authentication` are what exist without an access token, and will cache.

For this use case, where you want to allow either an authenticated or unauthenticated user access, but want to prefer to use the authenticated one if the user is logged in, we need to define it in _both_ files.

That's what led me to realize we were caching all requests, even non-GETs (although we don't really have any of those), in the unauthenticated loader path, resulting in this fix: https://github.com/artsy/metaphysics/pull/2107/files#diff-ab8e7dffb98da7ef74a8c8bbb9c214eeR110